scripts/provision-all

```diff
@@ -17,6 +17,11 @@ sudo qubesctl --show-output --skip-dom0 --targets sys-firewall state.sls sd-sys-
 echo "Set up dom0 config files, including RPC policies, and create VMs"
 sudo qubesctl --show-output state.highstate
 
+echo "Setup sd-log vm first"
+sudo qubesctl --show-output --skip-dom0 --targets sd-log state.sls sd-log-templates-files
```

Comment:
should be `sd-log-template-files` here

Reply:
I think the issue here is more than a typo (this is untested, merely suggesting):
```
sudo qubesctl --show-output --skip-dom0 --targets sd-log-template, sd-log state.highstate
```

I am getting an error after make clean that sd-log can't download the securedrop-log package (which makes sense because sd-log doesn't have internet connectivity, we need to apply state to `sd-log-buster-template`